Module not found: Default condition should be last one

### Issue description

I ran into "Module not found: Default condition should be last one" error on my console from importing `ChatPromptWrapper`.

### Expected Behavior

No compilation error.

### Actual Behavior

![スクリーンショット 2023-09-02 12 44 25](https://github.com/withcatai/node-llama-cpp/assets/101444904/970a92d2-1725-4c66-b3e2-597e2f2b5c58)

> Module not found: Default condition should be last one

### Steps to reproduce

1. Set up a Next.js project
2. Create a Route Handler, e.g. app/api/local/route.ts
3. Import `ChatPromptWrapper`

`import { ChatPromptWrapper } from 'node-llama-cpp';`

### My Environment

| Dependency               | Version  |
| ---                      | ---      |
| `next`         | 13.4.18         |
| `node-llama-cpp` | 2.1.2  |


### Additional Context

Next.js uses Webpack as its underlying bundler, and I suspect the ordering of default export in package.json matters for this specific bundler.

### Relevant Features Used

- [ ] Metal support
- [ ] CUDA support
- [ ] Grammar

### Are you willing to resolve this issue by submitting a Pull Request?

Yes, I have the time, but I don't know how to start. I would need guidance.

--- a/package.json
+++ b/package.json
@@ -18,22 +18,22 @@
   ],
   "exports": {
     ".": {
-      "default": "./dist/index.js",
       "import": "./dist/index.js",
       "node": "./dist/index.js",
-      "types": "./dist/index.d.ts"
+      "types": "./dist/index.d.ts",
+      "default": "./dist/index.js"
     },
     "./commands": {
-      "default": "./dist/commands.js",
       "import": "./dist/commands.js",
       "node": "./dist/commands.js",
-      "types": "./dist/commands.d.ts"
+      "types": "./dist/commands.d.ts",
+      "default": "./dist/commands.js"
     },
     "./commands.js": {
-      "default": "./dist/commands.js",
       "import": "./dist/commands.js",
       "node": "./dist/commands.js",
-      "types": "./dist/commands.d.ts"
+      "types": "./dist/commands.d.ts",
+      "default": "./dist/commands.js"
     }
   },
   "engines": {

